src/aind_behavior_curriculum/curriculum.py

```diff
@@ -1070,3 +1083,147 @@ def read_json(bucket_name: str, json_key: Path | str) -> dict:
         curr = cls.model_validate_json(json_string)
 
         return curr
+
+
+def create_curriculum(
+    name: str,
+    version: str,
+    *tasks: Type[Task],
```

Comment:
I personally dislike using *args like this - can tasks be a `List`?

Reply:
Ye I agree. This started off as a different method in my head but it has morphed into something a bit clunky. I will make it an `Iterable[Tasks]`

Reply:
Fixed in 6b5b32abe409ee8a2dcad66ead0b79f171198294. Hard to keep all PRs synched and running my tests. I will probably move everything to a single PR. Sorry for the sloppiness!